prow/spyglass/gcsartifact_fetcher_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package spyglass
+
+import (
+	"context"
+	"testing"
+)
+
+// Tests listing objects associated with the current job in GCS
+// TODO: fake-gcs-server does not support the GCS XML API, Issue #8923
+func testGCSListArtifacts(t *testing.T) {
```

Comment:
I thought tests needed to be exported?

Reply:
exported in latest, it was intentionally unexported because of the TODO